AppCenter/AppCenterTests/MSServiceNotificationDelegateTests.m

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSServiceNotificationDelegate.h"
+#import "MSTestFrameworks.h"
+
+@interface MSServiceNotificationDelegateTests : XCTestCase
+
+@end
+
+@implementation MSServiceNotificationDelegateTests
```

Comment:
You can probably add a test for when a delegate is registered but doesn't implement the optional method.

Reply:
There is no optional method in our serviceNotificationdelegate. There is only one required method. Please let me know if you there are other tests that we can add.